### PR TITLE
Update Scylla Manager to 3.2.5

### LINKED ISSUE #1588: Upgrade Manager - to 3.2.5
### What should the feature do?

It's important, as we have critical fixes in >3.2.

### What is the use case behind this feature?

Repair functionality, but also some backup/restore fixes.

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by tnozicka**
**Description of your changes:**
Updates Scylla Manager to `3.2.5`.

**Which issue is resolved by this Pull Request:**
Resolves #1588

### CODE REVIEW

File: test/e2e/set/scyllacluster/config.go [hunk 1]
```diff
@@ -6,9 +6,9 @@ import (
 
 const (
 	updateFromScyllaVersion  = "5.2.6"
```
Comment (zimnx) [inline test/e2e/set/scyllacluster/config.go:8]: `updateFromScyllaVersion` and `upgradeFromScyllaVersion` could be bumped to newer versions as well

**Review by zimnx — COMMENTED** [review]
one nit, lgtm for the rest

**Comment by tnozicka** [comment]
https://github.com/scylladb/scylla-operator/issues/1433#issuecomment-1832393797
/retest

**Comment by zimnx** [comment]
https://github.com/scylladb/scylla-operator/issues/1433#issuecomment-1832667077
/retest
/lgtm

**Comment by tnozicka** [comment]
green but we need to wait for official 3.2.5 release

**Comment by tnozicka** [comment]
manager 3.2.5 is out
/retest

**Review by zimnx — APPROVED** [review]
/lgtm

**Comment by zimnx** [comment]
Flake https://github.com/scylladb/scylla-operator/issues/1232#issuecomment-1852040846

/test e2e-gke-parallel

**Comment by zimnx** [comment]
Flake https://github.com/scylladb/scylla-operator/issues/1633

/test e2e-gke-parallel

**Comment by zimnx** [comment]
Manager E2E test failed, please validate whether it's related to update.

**Review by vponomaryov — APPROVED** [review]

**Comment by tnozicka** [comment]
@zimnx I didn't see anything obviously wrong with it and the test has succeeded in the next 2 consecutive runs, so I'd give it a shot and if it appears again

**Comment by zimnx** [comment]
Thanks.
/lgtm
/hold cancel

**Pull request merged**